apps/src/lib/node/ledger/ethereum_node/oracle.rs

```diff
@@ -3,7 +3,7 @@ use std::ops::Deref;
 use clarity::Address;
 use namada::types::ethereum_events::{EthAddress, EthereumEvent};
 use num256::Uint256;
-use tokio::sync::mpsc::UnboundedSender;
+use tokio::sync::mpsc::{Sender as BoundedSender};
```

Comment:
can we import it as `Sender`? This is what we do elsewhere in the codebase

Reply:
Yeah, this one has to stay. See the line directly below.

Reply:
o i c, we could import `oneshot::Sender` as `oneshot::Sender`, that could be clearer in the code anyway but think it's also fine as is then